run/markdown-preview/editor/requirements-test.txt

```
@@ -0,0 +1,2 @@
+pytest==5.3.2; python_version > "3.0"
+pytest==5.3.2; python_version < "3.0"
```

Only using Python 3.8, no need to clarify this any more. 
```suggestion
pytest==5.3.2
```